packages/agents/watcher/src/operations/validateAndSwitch.ts

```diff
@@ -1,49 +1,44 @@
-import { PauseResponse } from "@connext/nxtp-adapters-watcher";
+import { SwitchResponse, ReportEventType } from "@connext/nxtp-adapters-watcher";
 import { createMethodContext, RequestContext } from "@connext/nxtp-utils";
-
 import { getContext } from "../watcher";
 
-export const validateAndSwitch = async (requestContext: RequestContext, transactions?: Record<string, string[]>) => {
+export const validateAndSwitch = async (requestContext: RequestContext) => {
   const {
     adapters: { monitor },
+    logger,
   } = getContext();
-
-  const { needsSwitch, reason } = await monitor.validateProposal(requestContext);
-  if (needsSwitch) {
-    await switchAndAlert(requestContext, reason || "", transactions);
+  const { needsAction, reason } = await monitor.validateProposal(requestContext);
+  if (needsAction) {
+    await switchAndAlert(requestContext, reason);
+  } else if (reason) {
+    logger.info(reason);
   }
 };
 
-export const switchAndAlert = async (
-  requestContext: RequestContext,
-  reason: string,
-  transactions?: Record<string, string[]>,
-): Promise<PauseResponse[] | void> => {
+export const switchAndAlert = async (requestContext: RequestContext, reason: string = ""): Promise<SwitchResponse> => {
   const {
     adapters: { monitor },
     logger,
     config,
   } = getContext();
   const methodContext = createMethodContext(switchAndAlert.name);
 
-  const domains = Object.keys(config.chains);
-  logger.warn("SWITCHING TO SLOW MODE!!!", requestContext, methodContext, { reason, transactions });
+  const hubDomain = config.hubDomain;
+  logger.warn("SWITCHING TO SLOW MODE!!!", requestContext, methodContext, { reason, transactions: undefined });
```

Comment:
the `transactions` field will just be dropped here when its translated to logging services, and its not providing any particularly valuable information, so we should just drop it